1.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
This is here just in case something happens and someone else needs to run this script. Remove this script before checking in.